src/components/page-deployment/promote-deployment-action.js

```diff
@@ -1,35 +1,28 @@
 import deploymentModel from '../../models/deployment';
 import PropTypes from 'prop-types';
-import configHandler from '../../utils/config';
-import { keys as configKeys } from '../../utils/config/keys';
-import ActionsPage from '../actions-page';
-import LinkButton from '../link-button';
 import { routeWithParams } from '../../utils/string';
 import routes from '../../routes';
 import React from 'react';
+import { Button } from '@equinor/eds-core-react';
 
 const PromoteDeploymentAction = ({ appName, deploymentName, deployment }) => {
   return (
-    <span>
-      {deployment &&
-        configHandler.getConfig(configKeys.FLAGS).enablePromotionPipeline && (
-          <ActionsPage>
-            <LinkButton
-              to={routeWithParams(
-                routes.appJobNew,
-                { appName },
-                {
-                  pipeline: 'promote',
-                  deploymentName: deploymentName,
-                  fromEnvironment: deployment.environment,
-                }
-              )}
-            >
-              Promote deployment…
-            </LinkButton>
-          </ActionsPage>
+    <div>
+      <Button
+        href={routeWithParams(
```

Comment:
<Button> has the same navigation issue as Breadcrumb; full page reload.
Consider wrapping EDS Button component into existing LinkButton in web-console.

Reply:
wrapped EDS Button inside a NavLink instead.